components/edit/configure.tsx

```diff
@@ -219,49 +229,51 @@ const Configure: React.FC<ConfigureProps> = ({ collapsed }) => {
             >
               <div className="grid grid-cols-1 gap-2 w-full mb-2">
                 <div className="max-h-[30vh] flex flex-col space-y-2 overflow-auto">
-                  {Array.from(droppedFileDetails.entries()).map(
-                    (fileDetail, i) => (
-                      <div key={i} className="flex space-x-2">
-                        <div className="truncate w-full border-2 dark:border-zinc-700 text-sm pl-2 rounded-lg flex justify-between items-center">
-                          <div className="flex items-center space-x-2 overflow-x-auto">
-                            <RiFileSearchLine />
-                            <p className="capitalize">
+                  {Array.from(droppedFiles.entries()).map((fileDetail, i) => (
```

Comment:
nit: might be easier to read if you destructure the entries:
```suggestion
                  {Array.from(droppedFiles.entries()).map(([key, fileDetail], i) => (
```